plugins/standard_tests/newhelm/tests/safe.py

```diff
@@ -48,8 +49,9 @@ def get_metadata(self) -> TestMetadata:
 
     def get_dependencies(self) -> Mapping[str, ExternalData]:
         return {
-            "prompts_csv": WebData(
-                source_url=f"{self.data_source}/export?format=csv",
+            f"prompts_csv_{FOLDER_NAME}": GDriveData(
```

Comment:
I think we'll get additional clarity and flexibility if we make FOLDER_NAME something passed in to `__init__`. That way we could even have Tests using different versions of the data.

Reply:
I was thinking about doing that! But I thought it would be redundant with the `folder_url` arg.